.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '45 9 * * 3'
+  push:
+    branches: [ "main" ]
```

Comment:
We do many pushes (merging a PR) to main per week. Can we then do without the scheduled job?

Reply:
Coming back to this @harshitasao, any reason why we would not want to remove the `schedule` trigger and only keep branch protection rules and main?

Reply:
Agree with others on this. Let's remove the schedule, and run it on `push`.